dependencies.yaml

```diff
@@ -0,0 +1,49 @@
+---
+dependencies:
+  - name: go
+    version: 1.19
+    refPaths:
+      - path: .github/workflows/ci.yml
+        match: GO_VERSION
+      - path: .github/workflows/release.yml
+        match: GO_VERSION
+
+  - name: rust
+    version: 1.58.1
```

Comment:
Can we use a newer version of rust?

Reply:
I don't think so. Maybe @lsm5 has more input on when to expect new rust versions on RHEL / RHCOS 8/9.